pints/_mcmc/_global_ac.py

```diff
@@ -0,0 +1,85 @@
+#
+# Adaptive covariance MCMC method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class GlobalACMCMC(pints.GlobalAdaptiveCovarianceMCMC):
```

Comment:
Could we use a different name for this one, to distinguish it from the wider class of global methods? I'd be fine with using some nickname like we do with the Remi method

Reply:
Agreed. Changed it to 'SimpleACMCMC'